keps/sig-node/2043-pod-resource-concrete-assigments/kep.yaml

```diff
@@ -16,7 +16,8 @@ reviewers:
   - "@dashpole"
 approvers:
   - "@sig-node-leads"
-prr-approvers: []
+prr-approvers:
```

Comment:
you seem to be missing stage, milestone, etc.. See https://github.com/kubernetes/community/blob/master/sig-architecture/production-readiness.md#submitting-a-kep-for-production-readiness-approval

Reply:
it was here on 26th and 34th line, I also added latest-milesone